ktor-features/ktor-gson/jvm/src/io/ktor/gson/GsonSupport.kt

```diff
@@ -30,7 +31,7 @@ public class GsonConverter(private val gson: Gson = Gson()) : ContentConverter {
         return TextContent(gson.toJson(value), contentType.withCharset(context.call.suitableCharset()))
     }
 
-    override suspend fun convertForReceive(context: PipelineContext<ApplicationReceiveRequest, ApplicationCall>): Any? {
+    override suspend fun convertForReceive(context: ReceiveExecution): Any? {
```

Comment:
This name is absolutely confusing

Reply:
Please, refer to https://youtrack.jetbrains.com/issue/KTOR-929 and also I've updated a javadoc with descriptions. If you still don't like the name, let's discuss it here in comments and you can suggest any other, I don't have a strong opinion on how to name this thing